packages/reference/src/entries/WrappedEntryCard/FetchingWrappedEntryCard.tsx

```diff
@@ -132,20 +133,33 @@ export function FetchingWrappedEntryCard(props: EntryCardReferenceEditorProps) {
       onEdit,
       onRemove,
     };
+
+    const { hasCardEditActions, sdk } = props;
+
+    function renderDefaultCard(props?: CustomEntryCardProps) {
```

Comment:
Will this really still work if `props` is omitted or should they be not optional?

But yes :+1: looking good like this. Consider to update one of the integration tests using the `return renderDefaultCard(...props)` instead of `return false`, e.g. [here](https://github.com/contentful/field-editors/blob/fd1d9353d459f96f255430eba0f6b5217c30f973/packages/reference/src/entries/MultipleEntryReferenceEditor.mdx) in a follow-up PR.

Reply:
It can be optional, spreading `undefined` doesn't alter the object.

Reply:
Testcases added in `SingleEntryReferenceEditor.mdx` and `MultipleEntryReferenceEditor.mdx`.